docs/docs/overview.md

```diff
@@ -69,6 +69,22 @@ Create a production build which includes a `DockerFile` so that you can deploy a
 dart_frog build
 ```
 
+### Uninstalling 🗑️
```

Comment:
This is currently the section "Quick Start 🚀",  and I think it doesn't make much sense to have it there. Maybe:

```suggestion
## Uninstalling 🗑️
```